packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -103,6 +97,10 @@ export class CoinbaseWalletProvider extends EventEmitter implements ProviderInte
     });
   }
 
+  public get isCoinbaseWallet() {
+    return true;
+  }
```

Comment:
we could do something like
```suggestion
  readonly isCoinbaseWallet = true
```